RiotSwiftUI/Modules/Room/LocationSharing/LocationSharingScreenState.swift

```diff
@@ -35,7 +34,7 @@ enum MockLocationSharingScreenState: MockScreenState, CaseIterable {
             location = CLLocationCoordinate2D(latitude: 51.4932641, longitude: -0.257096)
         }
         
-        let mapURL = URL(string: "https://api.maptiler.com/maps/streets/style.json?key=" + RiotKeys().mapTilerAPIKey)!
+        let mapURL = URL(string: "https://api.maptiler.com/maps/streets/style.json?key=fU3vlMsMn4Jb6dnEIFsx")!
```

Comment:
We really need `BuildSettings` in the SwiftUI target. I've just hit that when trying to check a flag, will look into it.

Reply:
We definitely do, yes. Unfortunately BuildSettings has dependencies on strange things like `MXKRoomInputToolbarCompressionModePrompt`, `RoomTimelineStyleIdentifier` etc.